Makefile

```diff
@@ -28,7 +28,7 @@ unit-test:
 	go test -race ./... -tags=unitTest
 
 run-int-test:
-	PUBSUB_EMULATOR_HOST=localhost:8539 go test github.com/ONSdigital/census-rm-pubsub-adapter
+	PUBSUB_EMULATOR_HOST=localhost:8539 go test -count 1 github.com/ONSdigital/census-rm-pubsub-adapter
```

Comment:
The `-count 1` flag forces it to rerun the tests every time rather than skipping them if it's a cached a pass result and doesn't believe anything has changed.